.github/workflows/ci.yml

```diff
@@ -0,0 +1,18 @@
+name: CI workflow
+on: [push, pull_request]
```

Comment:
add master branch target?

Reply:
I think branch target should not be added because it will not trigger the check for dependent bot.

Reply:
dependabot will be triggered by the pull_request trigger. In this case you'll have double ci execution. One for the push on the branch another one on the pull request event.

Reply:
I do not think it will trigger twice, you can compare the setting across the fastify repositories. All of them do not have the branch target and also they do not trigger twice.
I don't think we should treat it special in this plugin.

Reply:
ok, so this bug has been fixed in GH actions.